Framework/Algorithms/src/CreateSampleWorkspace.cpp

```diff
@@ -159,6 +160,11 @@ void CreateSampleWorkspace::init() {
                   boost::make_shared<BoundedValidator<double>>(0, 1000, true),
                   "The distance along the beam direction from the source to "
                   "the sample in M (default:10.0)");
+  declareProperty("NumScanPoints", 1,
+                  boost::make_shared<BoundedValidator<int>>(0, 360, true),
```

Comment:
It probably would be better to be clear on this "1 degree increments" aspect of the functionality